solana/programs/example-native-token-transfers/src/instructions/release_inbound.rs

```diff
@@ -54,8 +54,8 @@ pub struct ReleaseInboundMint<'info> {
 }
 
 /// Release an inbound transfer and mint the tokens to the recipient.
-/// When `revert_on_error` is true, the transaction will revert if the
-/// release timestamp has not been reached. When `revert_on_error` is false, the
+/// When `revert_on_delay` is true, the transaction will revert if the
```

Comment:
The actual argument is called `revert_on_delay`. But perhaps this flag _should_ revert on _any_ error and not just the case where the transaction is not ready to be released?